Diz.Core/serialization/ProjectSerializer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Diagnostics;
+using System.IO;
+using Diz.Core.model;
+
+namespace Diz.Core.serialization
+{
+    abstract class ProjectSerializer
+    {
+        public const string Watermark = "DiztinGUIsh";
+
+        public abstract byte[] Save(Project project);
+        public abstract Project Load(byte[] data);
+
+        public void SaveToFile(Project project, string filename)
+        {
+            File.WriteAllBytes(filename, Save(project));
+        }
+        
+        public Project LoadFromFile(string filename)
+        {
+            return Load(File.ReadAllBytes(filename));
+        }
+
```

Comment:
Oh right, I need to disable this DebugVerifyProjectEquality() in non-debug builds.